src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -42,6 +49,10 @@
     <form action="/login" method="POST">
       <label for="username">Username: </label>
       <input type="text" name="username" id="username">
+      <br/>
+      <label for="password">Password: </label>
```

Comment:
Maybe instead of having `<br>` tags, we can have a CSS style on labels/inputs? Unsure what the convention is but I think that'll make it easier to restyle our website later on!

Reply:
This was code from Project 1, but if we need to make more changes in the future we can definitely put it in a CSS file!